.travis.yml

```diff
@@ -43,9 +43,11 @@ install:
 # of numpy
 #
   - pip install --no-deps javabridge
+  - pip install --no-deps pyzmq==13.1.0
```

Comment:
I think this was why the build was breaking on Travis:
https://travis-ci.org/CellProfiler/CellProfiler/builds/95658209

It failed in test_analysis which is using ZMQ, so I figured I'd revert. It worked, I do not know what I fixed. ZMQ 14+ works on windows, though.